lib/simulator_cuda.h

```diff
@@ -51,10 +51,21 @@ class SimulatorCUDA final {
 
   SimulatorCUDA() : scratch_(nullptr), scratch_size_(0) {
     ErrorCheck(cudaMalloc(&d_ws, max_buf_size));
+    d_ws_from_tf = false;
+  }
+
+  SimulatorCUDA(char* d_ws_temp) : scratch_(nullptr), scratch_size_(0) {
```

Comment:
Why do you need to inject this buffer?

Reply:
That's old code legacy, let me remove it :) I had to inject it from a temporary buffer from Tensorflow Quantum context before, but proved that it's useless if we don't use TensorFlow GPU context, and then we don't have any conflict between qsim GPU context and tensorflow.